src/Uno.UWPSyncGenerator/Generator.cs

```diff
@@ -1416,7 +1459,23 @@ private IMethodSymbol FindMatchingMethod(ITypeSymbol symbol, IMethodSymbol sourc
 			}
 		}
 
+		static Dictionary<(string projectFile, string targetFramework), Compilation> _projects
+			= new Dictionary<(string projectFile, string targetFramework), Compilation>();
+
 		private static Compilation LoadProject(string projectFile, string targetFramework = null)
+		{
+			var key = (projectFile, targetFramework);
+
+			if (_projects.TryGetValue(key, out var compilation))
+			{
+				Console.WriteLine($"Using cached compilation for {projectFile} and {targetFramework}");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5475653)